openedx_ledger/admin.py

```diff
@@ -354,3 +388,153 @@ def save_model(self, request, obj, form, change):
                 notes=obj.notes,
                 transaction_of_interest=obj.transaction_of_interest,
             )
+
+
+@admin.register(models.SalesContractReferenceProvider)
+class SalesContractReferenceProviderAdmin(SimpleHistoryAdmin):
+    """
+    Admin configuration for the SalesContractReferenceProvider model.
+    """
+    class Meta:
+        """
+        Metaclass for SalesContractReferenceProviderAdmin.
+        """
+
+        model = models.SalesContractReferenceProvider
+        fields = '__all__'
+
+    search_fields = ('name', 'slug',)
+    list_display = ('name', 'slug',)
+
+
+class DepositAdminCreateForm(forms.ModelForm):
+    """
+    Form that allows users to enter deposit quantities in dollars
+    instead of cents.
+    """
+    class Meta:
+        model = models.Deposit
+        fields = [
+            'ledger',
+            'quantity_usd_input',
+            'sales_contract_reference_id',
+            'sales_contract_reference_provider',
+        ]
+
+    quantity_usd_input = forms.FloatField(
+        required=True,
+        help_text=(
+            "Amount of deposit in US Dollars. Remember to increase the related policy spend limits to take advantage "
+            "of this new added value."
+        ),
```

Comment:
[suggestion/non-blocking] I would add some additional wording in here to highlight the recently added `spend_limit` constraints. 
Ex. "The increase in spend limit should not exceed the total deposits value."

As I am writing this out, this may not be the most appropriate place to add the help text addition I suggested, but open to feedback 👍🏽

Reply:
Oh also, its not clear whether it should be added in cents or usd (But I am guessing its going to be cents).

Reply:
I'm in favor of the help/warning message being closer to the policy admin page.  Description and field name clearly states USD.